.github/workflows/doxygen.yaml

```diff
@@ -0,0 +1,37 @@
+name: Deploy site
+
+on:
+  push:
+    branches: [ c3d-docs ]
```

Comment:
This would need to be updated to `master` before merging, correct?

Reply:
No. It would be changed to build on release.